src/components/views/context_menus/WidgetContextMenu.tsx

```diff
@@ -34,8 +35,10 @@ import { WidgetType } from "../../../widgets/WidgetType";
 import MatrixClientContext from "../../../contexts/MatrixClientContext";
 import { Container, WidgetLayoutStore } from "../../../stores/widgets/WidgetLayoutStore";
 import { getConfigLivestreamUrl, startJitsiAudioLivestream } from "../../../Livestream";
+import { ModuleRunner } from "../../../modules/ModuleRunner";
+import { ElementWidget } from "../../../stores/widgets/StopGapWidget";
 
-interface IProps extends React.ComponentProps<typeof IconizedContextMenu> {
+export interface WidgetContextMenuProps extends React.ComponentProps<typeof IconizedContextMenu> {
```

Comment:
We try to use `ComponentProps<typeof WidgetContextMenu>` if possible, instead of exporting props types.